src/cf-tables.less

```diff
@@ -391,21 +393,30 @@ table {
     }
```

Comment:
`sorted_down` and `sorted_up` need to be changed to use hyphens in the blocks above this line.